BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/physicsOfResultsPUEO.py

```diff
@@ -225,3 +261,38 @@ def getFiles(source, energy, indiv):
 cbar.set_ticklabels(['1', '10', '100', '1000', '10000', '100000'])
 fig5.savefig('{}/{}_maxEField_vs_maxEFieldFreq_2d_bestindiv.png'.format(g.destination, g.indiv))
 
+
+#Plot a histogram of the RF direction cosine theta
+fig7 = plt.figure()
+ax7 = fig7.add_subplot(111)
+N, bins, patches = ax7.hist(var_dict['RFdirCosTheta'], bins=100, range=(0, 1), align='mid', rwidth=0.8, color=plotting_colors[0])
+ax7.set_xlabel('RF Direction Cosine $\\theta$')
+ax7.set_ylabel('Number of Events')
+ax7.set_title('RF Direction Cosine $\\theta$ for {} EeV Neutrinos'.format(g.energy))
+ax7.grid(True)
+fig7.savefig('{}/{}_RFdirCosTheta_bestindiv.png'.format(g.destination, g.indiv))
```

Comment:
```suggestion
N, bins, patches = ax7.hist(var_dict['RFdirCosTheta'], bins=100, range=(0, 1),
                            align='mid', rwidth=0.8, color=plotting_colors[0])
ax7.set_xlabel('RF Direction Cosine $\\theta$')
ax7.set_ylabel('Number of Events')
ax7.set_title(f'RF Direction Cosine $\\theta$ for {g.energy} EeV Neutrinos')
ax7.grid(True)
fig7.savefig(f'{g.destination}/{g.indiv}_RFdirCosTheta_bestindiv.png')
```
This avoids long lines and uses f-strings.